Lib/unittest/mock.py

```diff
@@ -32,9 +33,11 @@
 import pprint
 import sys
 import builtins
+import threading
 from types import CodeType, ModuleType, MethodType
 from unittest.util import safe_repr
 from functools import wraps, partial
+from collections import defaultdict
```

Comment:
`defaultdict` is not used and can be removed. I started my original PR to store call object as key mapped to event but call objects are not always hashable.

Reply:
Wops, good catch, forgot to remove it in https://github.com/python/cpython/pull/16094/commits/28c3974d37b665a026af53c9e106c42db93541e9